modules/specialized-guides/pages/large-deployments/hub-online-sync.adoc

```diff
@@ -0,0 +1,213 @@
+
+//TO DO: What is the fundamental difference between these two operations? In both cases the connection between hub and periperal will be stopped. Can the invalidated token be re-validated?
+
+Be careful when using option btn:[Invalidate] as it no longer grants access to the other server.
+Invalidating the token can be undone, and token can be made valid again at any time.
+
+It is possible to delete a token.
+Deletion of the token can only occur when the token is not associated with a registered peripheral server or hub, and this action cannot be undone.
+
+
+=== Access hub server details from the peripheral server
+
+Every peripheral server stores the information about its hub server.
+
+[IMPORTANT]
+====
+A peripheral server can only have one hub server configured.
+====
+
+.Procedure: Accessing hub server details
+. On the peripheral server, go to menu:Hub Configuration[Hub Details].
+. On the screen [literal]``Hub Details`` find the information about the hub server.
+.. Field [literal]``Server FQDN`` shows the hub server's FQDN.
+.. Field [literal]``Registration date`` shows the time when the peripheral serevr was registsred to the hub server. 
+.. Field [literal]``Last modified`` shows the time of the last saved configuration change. 
+.. Field [literal]``Root Certificate Authority`` shows certificate details.
+   To download, edit or delete the root certificate, clicking btn:[Download], btn:[Edit] or btn:[Delete] respectively.
+   Deleting the certificate will break the connection between servers.
+.. Field [literal]``GPG Public Key`` shows whether the GPG key has been configured for the hub server.
+   For more information about GPG keys between hub and peripheral servers, see <<gpg-for-hub-online-sync>>.
+.. Field [literal]``Mirrir credentials`` is the username the peripheral server uses when connecting to the hub server to synchronize vendor channels.
+   This username is generated automatically on the hub server, and then transmitted to the peripheral server during the registration phase.
+
+
+[[gpg-for-hub-online-sync]]
+==== GPG key usage with hub online synhronization
+
+When the packages are signed with GPG key, the key is automatically transmitted from hub to peripheral server.
+
+By default, {productname} is not signing meta data.
+Therefore, when the pripheral server is downloading data from the hub server, unless the customer has created their own GPG key then they have no way of checking if the downloaded rpms have valid signature.
+
+To enable checking of the data integrity, the GPG key needs to be created on the hub side.
+When the peripheral server is configured to communicate with the hub, the public GPG key will then automatically be transferred to it.
+
+Once the GPG key is created on the hub, field [literal]``GPG Public Key`` will be set to show that this serevr is using the GPG key.
+For more information about setting up own GPG key, see xref:administration:repo-metadata.adoc[].
```

Comment:
I think you need to update the peripheral registration when you create the GPG key after you registered the peripheral.
I need to check how this can be done. Or Thomas knows it